src/dumpsxiso/main.cpp

```diff
@@ -425,6 +427,79 @@ std::unique_ptr<cd::IsoDirEntries> ParseSubdirectory(cd::IsoReader& reader, List
 	return dirEntries;
 }
 
+std::unique_ptr<cd::IsoDirEntries> ParsePathTable(cd::IsoReader& reader, ListView<cd::IsoDirEntries::Entry> view, std::vector<cd::IsoPathTable::Entry>& pathTableList, std::vector<cd::IsoPathTable::Entry>& sorted, int index,
+   const fs::path& path) {
+    auto dirEntries = std::make_unique<cd::IsoDirEntries>(std::move(view));
+
+		int sec = pathTableList[index].entry.extLength 
+				? pathTableList[index].entry.extLength 
+				: 1;		
```

Comment:
This is probably better done as `std::min(1, pathTableList[index].entry.extLength);`

Reply:
Pretty sure that wouldn't work, because 0 wouldn't read the directory (and I've found that in such obfuscated fs-es they leave it 0 sometimes)
If you want is as a one line `std::max(1, pathTableList[index].entry.extLength)` would work

Reply:
Of course, I got confused - I meant `max` indeed.